Gemfile

```diff
@@ -31,5 +31,11 @@ group :test do
   gem 'timecop'
 end
 
+group :development, :test do
+  # Pronto
+  gem 'pronto'
+  gem 'pronto-rubocop', require: false
```

Comment:
Bundler/DuplicatedGem: Gem `pronto-rubocop` requirements already given on line 26 of the Gemfile.